tools/modules/functions/check_os_status.sh

```diff
@@ -0,0 +1,42 @@
+
+module_options+=(
+    ["check_os_status,author"]="@Tearran"
+    ["check_os_status,feature"]="check_os_status"
+    ["check_os_status,example"]="help"
+    ["check_os_status,desc"]="Check if the current OS is supported based on /etc/armbian-distribution-status"
+    ["check_os_status,status"]="Active"
+)
+
+function check_os_status() {
+	case "$1" in
+		help)
+			echo "Usage: check_os_status"
+			echo "This function checks if the current OS is supported based on /etc/armbian-distribution-status."
+			echo "It retrieves the current OS distribution and checks if it is listed as supported in the specified file."
+			;;
+		*)
+			FILE="/etc/armbian-distribution-status"
+
+			# Detect the current OS distribution codename
+			CURRENT_OS=$(lsb_release -cs 2>/dev/null || grep "VERSION=" /etc/os-release | grep -oP '(?<=\().*(?=\))')
```

Comment:
Isn't this already known at this stage?
https://github.com/armbian/configng/blob/main/tools/modules/functions/set_runtime_variables.sh#L52-L62

Reply:
$DISTROID
```
No LSB modules are available.
bookworm
```
$CURRENT_OS
```
bookworm
```

Reply:
yeah, fix that first one. no need to execute this twice